firebase.json

```diff
@@ -1123,6 +1173,56 @@
           "destination": "https://zksync.io/contact",
           "type": "302"
         },
+        {
+          "source": "/docs/dev/building-on-zksync/hello-world.html",
```

Comment:
this is duplicate with the main branch (there are already two for prod and staging)